fxa-oauth-server/lib/encrypt.js

```diff
@@ -3,13 +3,10 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const crypto = require('crypto');
-
 const buf = require('buf').hex;
 
-const config = require('./config');
-
 exports.hash = function hash(value) {
-  var sha = crypto.createHash(config.get('encrypt.hashAlg'));
+  var sha = crypto.createHash('sha256');
```

Comment:
nit: `const HASH_ALGORITHM = 'sha256'; ...const sha = crypto.createHash(config.get(HASH_ALGORITH)`

Reply:
yeah it depends on person, some people prefer the const. others want the name in the funciton, i am not sure 🤷‍♂️